src/runtime/virtcontainers/hypervisor.go

```diff
@@ -591,7 +599,7 @@ func (conf *HypervisorConfig) AddCustomAsset(a *types.Asset) error {
 		return fmt.Errorf("Invalid %s at %s", a.Type(), a.Path())
 	}
 
-	virtLog.Debugf("Using custom %v asset %s", a.Type(), a.Path())
+	hvLogger.Debugf("Using custom %v asset %s", a.Type(), a.Path())
```

Comment:
Should this not be `hType.logger().` That I think is the way we use logger in other places.

See this example https://github.com/kata-containers/kata-containers/blob/main/src/runtime/virtcontainers/clh.go#L218

```
	clh.Logger().WithField("function", "CreateVM").Info("creating Sandbox")
```

Reply:
To start, I just matched what was existing before, but instead using our logger. Let's improve this in second pass as we prepare the hypervisors pkg?